osu.Game/Overlays/Settings/Sections/Gameplay/GeneralSettings.cs

```diff
@@ -33,6 +33,12 @@ private void load(OsuConfigManager config)
                     LabelText = GraphicsSettingsStrings.HitLighting,
                     Current = config.GetBindable<bool>(OsuSetting.HitLighting)
                 },
+                new SettingsCheckbox
+                {
+                    LabelText = UserInterfaceStrings.DisplaySpectatorList,
+                    Current = config.GetBindable<bool>(OsuSetting.DisplaySpectatorList),
+                    Keywords = new[] { "spectator", "display", "list" }
+                },
```

Comment:
Not sure about this being a setting rather than having the spectator list be a skin element that you can delete.

Reply:
Im kind of hesitant to agree, since its not technically a skin element (unless added), and more of a general visibility thing - I wouldnt want to change every skin to include or exclude this. From a player's perspective this is something not tied to any skin. Either I'd want it be there, or not.